TestAssets/NonRestoredTestProjects/DotnetAddProjProjects/Invalid.sln

```diff
@@ -0,0 +1 @@
+Testing an invalid solution file
```

Comment:
You want a newline? Is there a particular reason? The file is supposed to represent an invalid sln file...